docs/deploy.md

```diff
@@ -167,12 +167,15 @@ Specify which machines to deploy. For example (requires 5.5GB free memory):
 vagrant up m1 a1 p1 boot
 ```
 
+The order of startup is important: master VMs (beginning with `m`) must be started first.
```

Comment:
There's a whole section on constraints. Masters don't actually need to be first, they just need to be before boot.